lua/ulx/modules/sh/pcrestrict.lua

```diff
@@ -0,0 +1,45 @@
+CATEGORY_NAME = "CFC"
+
+local function addToWhitelist( callingPlayer, targetPlayers )
+    local affected_plys = {}
+
+    for _, ply in pairs( targetPlayers ) do
+        local isPlyWhitelisted = CFCPropcoreRestrict.playerIsWhitelisted( ply )
+
+        if isPlyWhitelisted then
+            ULib.tsayError( callingPlayer, ply:Name() .. " is already whitelisted!", true )
+        else
+            table.insert( affected_plys, ply )
+        end
+    end
+
+    CFCPropcoreRestrict.addPlayersToPropcoreWhitelist( affected_plys )
+    ulx.fancyLogAdmin( callingPlayer, true, "#A added #T to the propcore whitelist", affected_plys )
+end
+
+local PCWhitelistAdd = ulx.command( CATEGORY_NAME, "ulx allowpropcore", addToWhitelist, "!allowpropcore" )
+PCWhitelistAdd:addParam{ type = ULib.cmds.PlayersArg }
+PCWhitelistAdd:defaultAccess( ULib.ACCESS_ADMIN )
+PCWhitelistAdd:help( "Adds specified target(s) to a propcore whitelist" )
+
+local function removeFromWhitelist( callingPlayer, targetPlayers )
+    local affected_plys = {}
+
+    for _, ply in pairs( targetPlayers ) do
+        local isPlyWhitelisted = CFCPropcoreRestrict.playerIsWhitelisted( ply )
+
+        if not isPlyWhitelisted then
+            ULib.tsayError( callingPlayer, ply:Name() .. " is not whitelisted!", true )
+        else
+            table.insert( affected_plys, ply )
+        end
+    end
+
+    CFCPropcoreRestrict.removePlayersFromPropcoreWhitelist( affected_plys )
+    ulx.fancyLogAdmin( callingPlayer, true, "#A removed #T from the propcore whitelist", affected_plys )
+end
+
+local PCWhitelistRemove = ulx.command( CATEGORY_NAME, "ulx denypropcore", removeFromWhitelist, "!denypropcore" )
```

Comment:
i believe theres a way to inform ulx that a command is considered the opposite of another command
check out how [ulx gag](https://github.com/TeamUlysses/ulx/blob/8d41a87628a263058003c5f188aff6f86ffc255a/lua/ulx/modules/sh/chat.lua#L248)  does it